modules/@apostrophecms/soft-redirect/index.js

```diff
@@ -50,9 +51,9 @@ module.exports = {
           if (!(doc && doc._url)) {
             return;
           }
-          if (self.local(doc._url) !== req.url) {
+          if (doc._url !== url) {
```

Comment:
`url` won't contain the protocol and hostname, just the pathname, so we still need `self.local` here to make a valid comparison by eliminating the protocol and hostname from `doc._url` (if you don't see them it's because you don't have `baseUrl` set, but it's just about always set in production).

Reply:
I missed that. Thanks!